samples/databases/wide-world-importers/README.md

```diff
@@ -51,31 +54,42 @@ To run this sample, you need the following prerequisites.
 
 ## Sample structure
 
-The latest release of this sample is available here: TBD
+The latest release of this sample is available here: [wide-world-importers-release](http://go.microsoft.com/fwlink/?LinkID=800630)
+
+This sample contains databases as well as a number of sample scripts and workload drivers.
+
+The sample databases are created through SQL Server Data Tools projects in Visual Studio. Each database has its own project; the solution [wwi-sample.sln](wwi-sample.sln) in the root folder of the sample has references to all the projects.
```

Comment:
I am assuming we are still going to have those .bak files to download for the customers should they choose to not bother with SSDT?